src/rendergraph/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+find_package(QT NAMES Qt6 REQUIRED COMPONENTS Core)
+message(STATUS "Qt version ${QT_VERSION_MAJOR}.${QT_VERSION_MINOR}")
+
+if(QT_VERSION_MINOR GREATER_EQUAL 6)
+  set(USE_QSHADER_FOR_GL ON)
+endif()
+
+add_subdirectory(opengl)
+add_subdirectory(scenegraph)
+add_subdirectory(../../res/shaders/rendergraph shaders)
```

Comment:
Isn't using a parent directory kind of the opposite of what `add_subdirectory` is supposed to do?

Reply:
The alternative is to move the rendergraph shaders to src/rendergraph/shaders. I am not against that.